design/experimental-new-features-in-modular.md

```diff
@@ -0,0 +1,204 @@
+# Shipping experimental new features in Modular
+
+## Background
+
+As the Azure SDK for JavaScript continues to evolve, it is important for us to have a dedicated space to ship experimental features within our own client or language. By providing a separate subpath export for experimental features alongside with the regular release of Modular, we can gather valuable feedback from our customers who have tried these new features. This approach enables us to receive early insights and iterate on the experimental features while ensuring that our mainline releases remain stable and reliable.
+
+## Questions
+
+1. **Version picking strategy**  
+  One latest GA + One latest preview version.  
+  If the latest preview version < latest GA version, then we will use latest GA version as the experiment version.  
```

Comment:
why not remove the preview in this case?

Reply:
Agree with Jeremy. Removing the preview version is what we decided to do for Form Recognizer. We support all GA versions and _only the latest preview version _if and only if_ (1) the package is a beta version, in this case if it's experimental; and (2) the latest preview is _newer_ than the latest GA version. So we don't support old previews in any case.